README.md

```diff
@@ -9,3 +9,24 @@ No really, *watch it*... click the Watch button to stay on top of the latest cha
 
 ### :scroll: License
 As per usual, we are publishing under the [Apache 2 License](LICENSE).
+
+
+### Project Nutshell -- What we're exploring right now and where to find the work:

```

Comment:
This is a really long header! Could we shorten it and remove the colon? 

The first paragraph is a really good summary paragraph for the whole project, could we maybe move that to the top of the readme and change this to `Big concepts` or something.

Just an idea, I've seen other repos do something where they indicate if something is done with a checkbox, you could do it like this

The following is the big picture of the project and our way of keeping track of the epics and issues in this project as a snapshot with links to the key epics (and some issues).

```md
- [ ] [Pay It Backward Token](https://github.com/ConsenSys/web3studio-bootleg/issues/5)
- [ ] [Bootleg User Experience](https://github.com/ConsenSys/web3studio-bootleg/issues/9)
- [ ] Bootleg [DeveloperKit](https://github.com/ConsenSys/web3studio-bootleg/issues/12)
- [ ] Bootleg [Presentation to SXSW](https://github.com/ConsenSys/web3studio-bootleg/issues/8)
```
- [ ] [Pay It Backward Token](https://github.com/ConsenSys/web3studio-bootleg/issues/5)
- [ ] [Bootleg User Experience](https://github.com/ConsenSys/web3studio-bootleg/issues/9)
- [ ] Bootleg [DeveloperKit](https://github.com/ConsenSys/web3studio-bootleg/issues/12)
- [ ] Bootleg [Presentation to SXSW](https://github.com/ConsenSys/web3studio-bootleg/issues/8)